.github/workflows/release-on-push-to-main.template.yaml

```diff
@@ -65,8 +65,7 @@ jobs:
           push: true
           target: app
           tags: |
-            us-docker.pkg.dev/uwit-mci-iam/containers/${template:app_name}:${{ steps.get-version.outputs.version }}
-            us-docker.pkg.dev/uwit-mci-iam/containers/${template:app_name}:${{ env.DEPLOYMENT_ID }}
+            us-docker.pkg.dev/uwit-mci-iam/containers/${template:app_name_hyphen}:${{ steps.get-version.outputs.version }}
+            us-docker.pkg.dev/uwit-mci-iam/containers/${template:app_name_hyphen}:${{ env.DEPLOYMENT_ID }}
```

Comment:
Based on your comment in the body of the PR, didn't you intend to remove `DEPLOYMENT_ID` from this?

It will also need to be removed from the Dockerfile template.